quota/etcd/quotaapi/quota_server.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package quotaapi provides a Quota admin server implementation.
+package quotaapi
```

Comment:
I do prefer it to be in its own package, instead of just `quota/etcd`, if that's what you meant. We could do `quota/etcd/api`, but I felt that `api` was too generic. I'm open to suggestions.